static_src/components/app_list.jsx

```diff
@@ -0,0 +1,91 @@
+
+import React from 'react';
+import Reactable from 'reactable';
+
+import SpaceStore from '../stores/space_store.js';
+
+var Table = Reactable.Table,
+    unsafe = Reactable.unsafe;
+
+function stateSetter(props) {
+  var space = SpaceStore.get(props.initialSpaceGuid);
+  return {
+    apps: space && space.apps || [],
+    currentOrgGuid: props.initialOrgGuid,
+    currentSpaceGuid: props.initialSpaceGuid
+  }
+}
+
+export default class AppList extends React.Component {
```

Comment:
You have two Lists. This one and the service list. In case we make anymore lists, should he have a parent BaseList class to inherit from to help with extensibility?